util/dir/src/lib.rs

```diff
@@ -172,7 +177,9 @@ impl DatabaseDirectories {
 
 	/// DB root path, named after genesis hash
 	pub fn db_root_path(&self) -> PathBuf {
-		self.spec_root_path().join("db").join(format!("{:x}", H64::from(self.genesis_hash)))
+		// TODO(review): verify
```

Comment:
looks ok to me

Reply:
Actually, it seems to be incorrect: https://github.com/paritytech/primitives/compare/v0.4.2...v0.5.2#diff-db9881ebd53f06502f623b642f628295L98